src/components/Calendar/Calendar.tsx

```diff
@@ -49,15 +52,71 @@ const getViewsFromViewsProp = (views: View[]) => {
 }
 
 const Calendar = (props: Props) => {
-  const { view, views, events, disabled, onDateClick, onDateRangeSelected, onEventClick } = props
+  const {
+    view,
+    views,
+    events,
+    disabled,
+    onDateClick,
+    onDateRangeSelected,
+    onEventClick,
+    onPrevClick,
+    onNextClick,
+    onTodayClick,
+  } = props
   const fullCalendarRef = React.createRef<FullCalendar>()
+
+  const onNavClick = (to: 'prev' | 'next' | 'today') => {
+    const calendar = fullCalendarRef?.current?.getApi()
+    if (!calendar) {
+      return
+    }
+
+    const toCalls = {
+      prev: {
+        nav: () => calendar.prev(),
+        callback: onPrevClick,
+      },
+      next: {
+        nav: () => calendar.next(),
+        callback: onNextClick,
+      },
+      today: {
+        nav: () => calendar.today(),
+        callback: onTodayClick,
+      },
+    }
+
+    const { nav, callback } = toCalls[to]
+    nav()
+    if (callback !== undefined) {
+      callback()
+    }
+  }
+
   return (
     <FullCalendar
       events={events}
       ref={fullCalendarRef}
       selectable={!disabled}
+      customButtons={{
+        customPrev: {
+          text: 'Prev',
```

Comment:
I prefer spelling out the entire word. Thoughts?

Reply:
Agree, especially for users whose first language is not available in the app, and have to use English. Default is the icon arrows, but I think it'd be great to make it very clear 🙂

Reply:
✅ in 1f3b3d6